CesiumGltfReader/include/CesiumGltf/IExtensionJsonHandler.h

```diff
@@ -1,13 +1,14 @@
 #pragma once
 
 #include <CesiumJsonReader/IJsonHandler.h>
+#include <CesiumUtility/ExtensibleObject.h>
 
 #include <any>
 #include <string_view>
 
-namespace CesiumGltf {
+using namespace CesiumUtility;
 
-struct ExtensibleObject;
+namespace CesiumGltf {
```

Comment:
Couldn't use forward declarations anymore after the changes in `ExtensibleObject.h`